Phoenix/Core/Include/Core/Voxels/Blocks.hpp

```diff
@@ -54,21 +56,22 @@ namespace phoenix
 		};
 
 		/// @brief Stores universal block definitions
-		class BlockRegistry
+		class BlockRegistry : public qz::utils::Singleton<BlockRegistry>
 		{
 		private:
 			std::vector<RegisteredBlock> Blocks;
```

Comment:
Convention for member variables is `m_camelCaseWithPrefix`
```suggestion
			std::vector<RegisteredBlock> m_blocks;
```